playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
+    
+        if (idx == 1 || idx == 2) {
+            return 1
+        } else if( idx >= 3) {
```

Comment:
```suggestion
        if (idx == 1 || idx == 2) {
            return 1
        }
        
        if( idx >= 3) {
```

이렇게 return 이 포함된 경우에는 이렇게 처리해주기도 합니다. early return 패턴이라고 합니다.

Reply:
```suggestion
        if (idx == 1 || idx == 2) {
            return 1
        } else if ( idx >= 3) {
```